### PR TITLE
[IMPROVE] add hover effect 

### PR DESCRIPTION
**Pull request by Deepak-learner**
Added hover effect for changing the colour of the icon and link on MessageBox in omni channel Livechat.

Additional context :

https://user-images.githubusercontent.com/56799414/111767984-5a531680-88cd-11eb-8695-9abb526fc3cb.mov

Issue solve:
#565

### CODE REVIEW

**Comment by Deepak-learner** [comment]
@rafaelblink , @murtaza98  please review.

File: src/routes/Chat/component.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ export default class Chat extends Component {
 							handleEmojiClick={this.handleEmojiClick}
 							pre={(
 								<ComposerActions>
-									<ComposerAction onClick={this.toggleEmojiPickerState}>
+									<ComposerAction className={createClassName(styles, 'emoji')} onClick={this.toggleEmojiPickerState}>
```
Comment (murtaza98) [inline src/routes/Chat/component.js:202]:
```suggestion
									<ComposerAction className={createClassName(styles, 'emoji-picker-icon')} onClick={this.toggleEmojiPickerState}>
```

Comment (murtaza98) [reply src/routes/Chat/component.js:202]: The class name was too generic earlier

File: src/routes/Chat/styles.scss [hunk 2]
```diff
@@ -56,6 +56,14 @@
 
 		opacity: 0.6;
 	}
+
+	.emoji{
```
Comment (murtaza98) [inline src/routes/Chat/styles.scss:60]:
```suggestion
	.emoji-picker-icon{
```

**Comment by Deepak-learner** [comment]
@murtaza98  thanks for your suggestion , I make the necessary changes , please review.

**Comment by murtaza98** [comment]
LGTM!

@renatobecker This PR looks good to be merged

**Review by murtaza98 — APPROVED** [review]

**Pull request merged**